src/service/navigation.js

```diff
@@ -177,13 +178,25 @@ export class Navigation {
    * @param {!Window} win
    * @param {string} url
    * @param {string=} opt_requestedBy
+   * @param {string=} opt_target
    */
-  navigateTo(win, url, opt_requestedBy) {
+  navigateTo(win, url, opt_requestedBy, opt_target) {
     const urlService = Services.urlForDoc(this.ampdoc);
     if (!urlService.isProtocolValid(url)) {
       user().error(TAG, 'Cannot navigate to invalid protocol: ' + url);
       return;
     }
+    
+    user().assert(
+        TAG, !opt_target || VALID_TARGETS.includes(opt_target),
+        `Target '${opt_target}' not supported.`);
+    
+    // If we have a target of "_blank", we will want to open a new window. A target
+    // of "_self" should behave like it would on an anchor tag and update the URL.
+    if (opt_target == '_blank') {
+      win.top.open(url, opt_target);
```

Comment:
I think we will need to handle the equivalent of `noreferrer` and `nofollow` here (maybe invert this, and make people specify if they want follow/referrer). We can do `nofollow` by doing:

```javascript
const newWin = win.top.open(url, opt_target);
newWin.opener = null;
```

This will prevent the new window from having a reference to our window (and say navigating it to another page).

Dealing with the referrer is a bit more tricky to do, but worth looking into.

Reply:
`noopener` also works for some browsers. https://developer.mozilla.org/en-US/docs/Web/API/Window/open#Window_functionality_features

Reply:
While that works, it also forces a new window in Chrome (desktop) that has no tab bar instead of opening a new tab.

It seems like setting `newWin.opener = null` might not work in Safari.